internal/gok/set.go

```diff
@@ -0,0 +1,72 @@
+package gok
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/antihax/optional"
+	"github.com/gokrazy/gokapi/gusapi"
+	"github.com/gokrazy/internal/instanceflag"
+	"github.com/spf13/cobra"
+)
+
+// setCmd is gok gus set.
+var setCmd = &cobra.Command{
+	// GroupID: "server",
+	Use:   "set",
+	Short: "Set the desired SBOM version for a machineID patter on the remote GUS server",
```

Comment:
typo: patter → pattern (also in the next line)

Reply:
Thanks, fixed.